modules/data-loading/pages/streaming-data-connector.adoc

```diff
@@ -0,0 +1,438 @@
+file.reader.settings.fs.gs.auth.service.account.enable=true
+file.reader.settings.fs.gs.impl=com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem
+file.reader.settings.fs.AbstractFileSystem.gs.impl="com.google.cloud.hadoop.fs.gcs.GoogleHadoopFS"
+
+mode=eof
+file.regexp=".*"
+file.recursive=true
+file.reader.type=text
+file.reader.batch.size=10000
+file.reader.text.eol="\\n"
+file.reader.text.header=true
+file.reader.text.archive.type=auto
+file.reader.text.archive.extensions.tar=tar
+file.reader.text.archive.extensions.zip=zip
+file.reader.text.archive.extensions.gzip=tar.gz,tgz
+
+[connector_person]
+name = fs-person-demo-104
+tasks.max=10
+topic=person-demo-104
+file.uris=gs://tg_csv/p.csv
+
+[connector_friend]
+name = fs-friend-demo-104
+tasks.max=10
+topic=friend-demo-104
+file.uris=gs://tg_csv/f.csv
+----
+
+==== Create connector
+Run command `gadmin connector create` and specify the configuration file to create the connector:
+
+  gadmin connector create --c <config_file>
+
+The connectors start streaming from the data source immediately after creation if the configurations are valid.
+You can run `gadmin connector status` to verify the status of the connectors.
+If the configurations are valid, the connectors should be in `RUNNING` status.
+
+Data streamed from the source will stay in TigerGraph's internal Kafka topics until they are ingested by a loading job.
+
+==== Create data source
+Now that the connector has started loading data into TigerGraph's internal Kafka cluster, you can create a data source and point it to the Kafka cluster:
+
+. Create a data source configuration file.
+The broker's IP and hostname should be `localhost:30002`, which points to the port for TigerGraph's internal Kafka cluster.
+In the `kafka.config` field, set `group.id` to `tigergraph`:
++
+[,json]
+----
+{
+	"broker":"localhost:30002",
+    "kafka_config":
+        {
+            "group.id": "tigergraph"
+        }
+}
+----
+. Run `CREATE DATA SOURCE` to create the data source:
++
+[,gsql]
+----
+CREATE DATA_SOURCE KAFKA k1 FOR GRAPH social
+----
+
+==== Create loading job
+Create a loading job to load data from the data source:
+
+. Create a topic-partition configuration for each topic.
++
+[,javascript]
+----
+{
+  "topic": <topic_name>, <1>
+  "partition_list": [ <2>
+    {
+      "start_offset": <offset_value>, <3>
+      "partition": <partition_number> <4>
+    },
+    {
+      "start_offset": <offset_value>, <3>
+      "partition": <partition_number> <4>
+    }
+    ...
+  ]
+}
+----
+<1> Replace `<topic_name>` with the name of the topic this configuration applies to.
+<2> List of partitions you want to stream from.
+For each partition, you can set a start offset.
+If this list is empty, or `partition_list` isn't included, all partitions are used with the default offset.
+<3> Replace `<offset_value>` with the offset value you want.
+The default offset for loading is `-1`, which means you will load data from the most recent message in the topic.
```

Comment:
For now, if users want to use -1, they need to remove the offset settings and let it run with the default settings, but this will be fixed later.